images/README.md

```diff
@@ -5,7 +5,7 @@ This is used for distribution of RDMA CNI binary in a Docker image.
 Typically you'd build this from the root of your RDMA CNI clone, and you'd set the `DOCKERFILE` to specify the Dockerfile during build time, `TAG` to specify the image's tag:
 
 ```
-$ DOCKERFILE=Dockerfile TAG=Mellanox/rdma-cni make image
+$ DOCKERFILE=Dockerfile TAG=mellanox/rdma-cni make image
```

Comment:
is rdma or rdma-cni. Sometimes  is rdma and sometime rdma-cni

Reply:
rdma-cni - is the image name
rdma - is the binary name